tfx/components/trainer/component.py

```diff
@@ -198,8 +200,8 @@ def trainer_fn(trainer.executor.TrainerFnArgs,
     """
     if [bool(module_file), bool(run_fn), bool(trainer_fn)].count(True) != 1:
       raise ValueError(
-          "Exactly one of 'module_file', 'trainer_fn', or 'run_fn' must be supplied."
-      )
+          "Exactly one of 'module_file', 'trainer_fn', or 'run_fn' must be "
+          "supplied.")
```

Comment:
why change this?

Reply:
this line throws a pylint error in the previous commits